src/app/search/projects/page.tsx

```diff
@@ -1,29 +1,35 @@
 'use client'
-import { NextPage } from 'next'
 import { useEffect, useLayoutEffect } from 'react'
-import { Container, Flex, Box, Button, Hide } from '@chakra-ui/react'
-import { shouldReposition, setScroll } from '@/utils/Stickify'
+import { NextPage } from 'next'
+import { useSearchParams } from 'next/navigation'
+import { Box, Button, Container, Flex, Hide } from '@chakra-ui/react'
+
+import { useActions, useAppState, useEffects } from '@/overmind'
+import { ALLOWED_RENDER_TYPE } from '@/@types/ProjectSearchResult'
```

Comment:
What is ALLOWED_RENDER_TYPE?

Reply:
ALLOWED_RENDER_TYPE is used to throw an error if the table with the "stickify" scrolling is rendered in a component that is not allowed. This stickify functionality is used predominantly in the project search result for now and I believe it was put in place as after a discussion be you and Mederic previously